Sources/BraintreeCoreSwift/BTClientTokenError.swift

```diff
@@ -0,0 +1,53 @@
+import Foundation
+
+///  Error codes associated with a client token.
+@objc public enum BTClientTokenError: Int, Error, CustomNSError, LocalizedError {
+
+    /// Authorization fingerprint was not present or invalid
+    case invalidAuthorizationFingerprint
+
+    /// Config URL was missing or invalid
+    case invalidConfigURL
+
+    /// Invalid JSON
+    case invalidJSON
+
+    /// Invalid client token format
+    case invalidFormat
+
+    /// Unsupported client token version
+    case unsupportedVersion
+    
+    /// Invalid UTF8 Encoding
+    case expectedUTF8Encoding
+    
+    /// Invalid Base64 Encoding
+    case expectedBase64Encoding
```

Comment:
Add new error names & corresponding code int value to the CHANGELOG

Reply:
Updated the CHANGELOG but since several error enums have been updated kept it slightly more vague vs calling out every error and its new value. Thoughts on the update or should we be even more explicit?

Reply:
I'm always in favor of an explicit CHANGELOG. I think it's also really helpful for us to remember when we did things. _Especially_ with our converting to Swift, it will be really hard to track actual implementation changes/history since this will all be entirely new commit history from our ObjC SDK.

So - maybe something like:
```
  * BraintreeCore
    * Remove `BTAPIClientErrorDomain` global constant
    * Add `BTClientTokenError.expectedBase64Encoding` error type 
    * Add `BTClientTokenError.expectedUTF8Encoding ` error type 
    ...
```